confluent_platform/tests/metrics.py

```diff
@@ -7,7 +7,6 @@
     'confluent.kafka.cluster.partition.under_min_isr',
     'confluent.kafka.controller.active_controller_count',
     'confluent.kafka.controller.leader_election_rate_and_time_ms.avg',
-    'confluent.kafka.controller.leader_election_rate_and_time_ms.rate',
```

Comment:
A bit suprised that this isn't a bit more deterministic considering the `*.avg` variant didn't fail.